src/commonMain/kotlin/software/momento/kotlin/sdk/responses/cache/control/CacheInfo.kt

```diff
@@ -0,0 +1,18 @@
+package software.momento.kotlin.sdk.responses.cache.control
+/** Information about a cache.  */
+public class CacheInfo
```

Comment:
This can be a data class, and `name` can be a `public val` so it doesn't need a getter.